### ISSUE TITLE
Add templates to create a pandoc-ling compatible version of the doc

### ISSUE THREAD

**Issue by xrotwang** [cldf-datasets/lgr#4]
See https://github.com/cysouw/pandoc-ling

Use this to create a PDF!?

**Comment by fmatter** [comment]
Happy to report that I have a pylingdocs version of this document, which comes with a PDF version :

[lgr-pld.pdf](https://github.com/cldf-datasets/lgr/files/13573036/lgr-pld.pdf)

**Comment by xrotwang** [comment]
@fmatter very cool! Can you point me to the sources so I could incorporate and run your pipeline as an example here, too?

**Comment by fmatter** [comment]
Yes I haven't taken the time to make it public yet, I'll try to get round to it this week.
